src/main/scala/org/broadinstitute/dsde/agora/server/SwaggerRoutes.scala

```diff
@@ -31,8 +29,8 @@ trait SwaggerRoutes {
       // want to catch all paths lest we circumvent Spray's not-found and method-not-allowed error
       // messages.
       (pathSuffixTest("o2c.html") | pathSuffixTest("swagger-ui.js")
-        | pathPrefixTest("css" /) | pathPrefixTest("fonts" /) | pathPrefixTest("images" /)
-        | pathPrefixTest("lang" /) | pathPrefixTest("lib" /)) {
+        | pathPrefixTest("css") | pathPrefixTest("fonts") | pathPrefixTest("images")
+        | pathPrefixTest("lang") | pathPrefixTest("lib")) {
```

Comment:
the warning about `postfixOps` was due to these trailing slashes ... which aren't necessary afaict. Instead of fixing the warning by importing `postfixOps`, I fixed the warning by removing the slashes.